### PR TITLE
add onLoading* prop for WebView

### PR DESCRIPTION
**Pull request by magicismight**
add onLoadingStart,onLoadingError,onLoadingFinish prop for WebView

### CODE REVIEW

**Comment by nicklockwood** [comment]
Looks good, but can you follow the same naming convention for these props as we use for `<Image>` ?

onLoadStart // same as your onLoadingStart
onError // same as your onLoadingError
onLoad // same as your onLoadingFinish
onLoadEnd // fires when loading ends either successfully or due to an error

File: Libraries/Components/WebView/WebView.android.js [hunk 1]
```diff
@@ -189,11 +192,19 @@ var WebView = React.createClass({
   },
 
   onLoadingStart: function(event) {
+    var onLoadingStart = this.props.onLoadingStart;
+    if (onLoadingStart) {
```
Comment (nicklockwood) [inline Libraries/Components/WebView/WebView.android.js]: We tend to use this shorthand in our codebase:

    var onLoadingStart = this.props.onLoadingStart;
    onLoadingStart && onLoadingStart(event)

Comment (magicismight) [inline Libraries/Components/WebView/WebView.android.js]: OK.
I`ll send an other PR with this fix later

File: Libraries/Components/WebView/WebView.ios.js [hunk 2]
```diff
@@ -98,6 +98,22 @@ var WebView = React.createClass({
      */
     renderLoading: PropTypes.func,
     /**
+     * Invoked when load finish
+     */
```
Comment (nicklockwood) [inline Libraries/Components/WebView/WebView.ios.js]: These errors seem spurious. @vjeux?

**Comment by nicklockwood** [comment]
@facebook-github-bot shipit

File: Libraries/Components/WebView/WebView.ios.js (hunk 2 above)
Comment (vjeux) [inline Libraries/Components/WebView/WebView.ios.js]: Yeah i'm not really sure what's going on here

Comment (magicismight) [inline Libraries/Components/WebView/WebView.ios.js]: a commit before have these errors,but fixed in the next commit.
my apology

Comment (vjeux) [inline Libraries/Components/WebView/WebView.ios.js]: Oh yeah, it's github being annoying by still showing outdated comments. I was thinking of deleting all the previous comments when receiving a new update to improve the situation

**Pull request closed without merging**